nextflow.config

```
@@ -24,6 +25,7 @@ params{
   gtf           = 's3://nextflow-ccdl-data/reference/homo_sapiens/ensembl-104/annotation/Homo_sapiens.GRCh38.104.gtf.gz'
   index_path    = 's3://nextflow-ccdl-data/reference/homo_sapiens/ensembl-104/salmon_index/Homo_sapiens.GRCh38.104.spliced_intron.txome'
   bulk_index    = 's3://nextflow-ccdl-data/reference/homo_sapiens/ensembl-104/salmon_index/Homo_sapiens.GRCh38.104.spliced_cdna.txome'
+  spaceranger_index = 's3://nextflow-ccdl-data/reference/homo_sapiens/ensembl-100/cellranger_index/cdna'
```

One minor thing, one more major:

Should this be `ensembl104`? I don't know if you used that for the index or if we might need to make another index, but we should be on the same version for everything.

```suggestion
  spaceranger_index = 's3://nextflow-ccdl-data/reference/homo_sapiens/ensembl-104/cellranger_index/cdna'
```

Separately, we should probably file an issue to add this index creation to `build-index.nf` 

The more minor thing is just that things were so nicely aligned before, it would be nice to maintain that. It will mean changing more lines, but that is okay...

---

We need to make another index that uses ensembl 104 first before we change this, so I will add a separate issue that adds building the cellranger index to `build-index.nf` and then when we do that we can update this.